purchase_order_type_product_categ/models/purchase_order_type.py

```diff
@@ -0,0 +1,10 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+
+class PurchaseOrderType(models.Model):
+    _inherit = "purchase.order.type"
+
+    product_category_id = fields.Many2one('product.category')
```

Comment:
Add a `help` attribute with full description and usage for this field. E.g.  "if filled, orders for this type will only be able to select products that belong to this category hierarchy"